### ISSUE TITLE
[Question] How to convert a .nemo checkpoint to onnx 

### ISSUE THREAD

**Issue by Amels404** [NVIDIA/NeMo#1962]
**Describe your question**

How to convert .nemo ckpt to onnx? 
I've tried to convert a .nemo model trained on 1.0.0b1 version of nemo to onnx. 
I want to use the onnx for deployment using Triton later on. 

**Environment overview **

 - Environment location: Conda 
 - Method of NeMo install: pip install== 1.0.0b3 

**Environment details**

- OS version: ubuntu 18.04 
- Torch 1.7.1
- Python 3.6.13

**Additional context**
 The model was trained on Arabic langauge. 
 I've tried to use this file: https://github.com/NVIDIA/NeMo/blob/v1.0.0b3/scripts/convasr_to_single_onnx.py
 and got this error: 

![image](https://user-images.githubusercontent.com/71659848/112324884-81d22680-8cb3-11eb-8843-ced825715353.png)

Thank you for your time!

**Comment by okuchaiev** [comment]
Can you try installing from main branch the most recent version?

**Comment by Amels404** [comment]
Thank you for your prompt response! 
I've tried to installed it like this: 
apt-get update && apt-get install -y libsndfile1 ffmpeg
pip install Cython
pip install nemo_toolkit[all]==1.0.0b3

and I got another error related to writing the onnx file. 

![image](https://user-images.githubusercontent.com/71659848/112643590-ce993700-8e44-11eb-99db-46573036144b.png)

I'm not sure if this is what you mean by installing from main but I've tried,also, installing from the main git branch and I got an error in the nemo import related to pyramid installation:    
 from pyramid.session import UnencryptedCookieSessionFactoryConfig                                                                                                                                                                        ImportError: cannot import name 'UnencryptedCookieSessionFactoryConfig' from 'pyramid.session' (unknown location) 
 
 Thank you again.
